src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -1754,18 +1754,25 @@ await ServiceHostInstance.SendEvent(
                 identifiers.Add(identifier.ToString());
             }
 
+            int count = 0;
+            string[] nonTSqlKeywords = new string[2];
```

Comment:
Use a shared const for the threshold to make it easier in case we change it again.